src/source/Crypto/Dtls_mbedtls.c

```diff
@@ -598,6 +672,29 @@ STATUS copyCertificateAndKey(mbedtls_x509_crt* pCert, mbedtls_pk_context* pKey,
     return retStatus;
 }
 
+#if !(defined(MBEDTLS_BIGNUM_C) && !defined(MBEDTLS_DEPRECATED_REMOVED))
+int mbedtls_x509write_crt_set_serial(mbedtls_x509write_cert* ctx, const mbedtls_mpi* serial)
```

Comment:
Just checking, it seems they removed this function: https://github.com/Mbed-TLS/mbedtls/issues/9892, but seems like it wasn't removed in the version specified here, 3.6.0

Reply:
I see, I think this still should hold valid. Maybe we can use `mbedtls_x509write_crt_set_serial_raw` in some other change. But it was lot of code change in the current file so I chose to simply make the deprecated function available for us.

To add the a different discussion to this: I am setting the mbedtls version to `v3.6` instead of `v3.6.0` so we always pull the latest patch version of mbedtls which keep addressing the bug fixes and any other minor issues related to mbedtls.